packages/framework/src/Model/Mail/Mailer.php

```diff
@@ -99,45 +74,37 @@ protected function getMessageWithReplacedVariables(MessageData $messageData)
             $messageData->subject,
             $messageData->variablesReplacementsForSubject
         );
-        $fromEmail = $messageData->fromEmail;
-        $fromName = $messageData->fromName;
 
-        $message = new Swift_Message();
-        $message->setSubject($subject);
-        $message->setFrom($fromEmail, $fromName);
-        $message->setTo($toEmail);
+        $email = new Email();
+        $email
+            ->subject($subject)
+            ->from(new Address($messageData->fromEmail, $messageData->fromName))
+            ->to($messageData->toEmail)
+            ->text(htmlspecialchars_decode(strip_tags($body), ENT_QUOTES))
+            ->html($body);
         if ($messageData->bccEmail !== null) {
-            $message->addBcc($messageData->bccEmail);
+            $email->addBcc($messageData->bccEmail);
         }
         if ($messageData->replyTo !== null) {
-            $message->addReplyTo($messageData->replyTo);
+            $email->addReplyTo($messageData->replyTo);
         }
-        $message->setContentType('text/plain; charset=UTF-8');
-        $message->setBody(htmlspecialchars_decode(strip_tags($body), ENT_QUOTES), 'text/plain');
-        $message->addPart($body, 'text/html');
 
         foreach ($messageData->attachments as $attachment) {
-            try {
-                $swiftAttachment = Swift_Attachment::fromPath(
-                    $this->mailTemplateFacade->getMailTemplateAttachmentFilepath($attachment)
-                );
-            } catch (FileNotFoundException $exception) {
-                $this->logger->error('Attachment could not be added because file was not found.', [$exception]);
-                continue;
-            }
-            $swiftAttachment->setFilename($attachment->getNameWithExtension());
-            $message->attach($swiftAttachment);
+            $email->attachFromPath(
+                $this->mailTemplateFacade->getMailTemplateAttachmentFilepath($attachment),
+                $attachment->getNameWithExtension()
```

Comment:
also the path for the temporary files seems weird – `project-base/src/var/fileUploads`
it was created after I have added a new attachments to the mail template

Reply:
yeah, it is super-weird, I will try it on master and create a separate issue for this eventually

Reply:
How did it go?

Reply:
I have not gotten to that yet :frowning_face:

Reply:
I am trying it now and I am not able to reproduce it, neither on master nor on this branch :thinking: I do not want to block this task furthermore, so I will merge this one, and let's connect after your vacation and check the behavior once more :wink: